src/Contextually/Relevant.InfoAutoAddedToExceptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿#if !NETSTANDARD1_3
```

Comment:
Let's drop 1.3 support in favor of 2.0 and remove this

Reply:
Why then to support Fx 4.5.2?
P.S. that would require major version update.